indico/modules/rb_new/client/js/common/bookings/BookingEditForm.jsx

```diff
@@ -207,36 +207,41 @@ class BookingEditForm extends React.Component {
         const bookedByCurrentUser = sessionUser.id === bookedForUser.id;
         const today = moment();
         const bookingFinished = today.isAfter(startDt, 'day') && (endDt && today.isAfter(endDt, 'day'));
-
+        const onlyDaily = hideOptions.single && hideOptions.recurring && !hideOptions.daily;
```

Comment:
shouldn't this check for all possible combinations where only one option is visible?

Reply:
In the code for the Landing page the logic is a bit different and we only check for `onylDaily`, so I wasn't sure what kind of approach I should follow.

Reply:
I guess we should change it there as well then :)

Reply:
Discard the prop `onlyDaily` and replace it with `hideOptions.single` etc?

Reply:
yep